packages/wp-dashboard/src/components/editorSettings/googleAnalytics/index.js

```diff
@@ -247,97 +291,112 @@ function GoogleAnalyticsSettings({
         )}
       </div>
       <div>
-        {analyticsActive ? (
-          <TextInputHelperText size={TextSize.Small}>
-            {TEXT.SITE_KIT_IN_USE}
-          </TextInputHelperText>
-        ) : (
+        <InlineForm>
+          <VisuallyHiddenLabel htmlFor="gaTrackingId">
+            {TEXT.ARIA_LABEL}
+          </VisuallyHiddenLabel>
+          <SettingsTextInput
+            aria-label={TEXT.ARIA_LABEL}
+            id="gaTrackingId"
+            value={analyticsId}
+            onChange={onUpdateAnalyticsId}
+            onKeyDown={handleOnKeyDown}
+            placeholder={TEXT.PLACEHOLDER}
+            hasError={Boolean(inputError)}
+            hint={inputError}
+            disabled={analyticsActive}
+          />
+          <SaveButton
+            type={ButtonType.Secondary}
+            size={ButtonSize.Small}
+            disabled={disableSaveButton}
+            onClick={handleOnSave}
+          >
+            {TEXT.SUBMIT_BUTTON}
+          </SaveButton>
+        </InlineForm>
+        <TextInputHelperText size={TextSize.Small}>
+          <TranslateWithMarkup
+            mapping={{
+              a: (
+                <InlineLink
+                  href={TEXT.CONTEXT_LINK}
+                  rel="noreferrer"
+                  target="_blank"
+                  size={TextSize.Small}
+                  onClick={onContextClick}
+                />
+              ),
+            }}
+          >
+            {TEXT.CONTEXT}
+          </TranslateWithMarkup>
+        </TextInputHelperText>
+        {analyticsActive && (
           <>
-            <InlineForm>
-              <VisuallyHiddenLabel htmlFor="gaTrackingId">
-                {TEXT.ARIA_LABEL}
-              </VisuallyHiddenLabel>
-              <SettingsTextInput
-                aria-label={TEXT.ARIA_LABEL}
-                id="gaTrackingId"
-                value={analyticsId}
-                onChange={onUpdateAnalyticsId}
-                onKeyDown={handleOnKeyDown}
-                placeholder={TEXT.PLACEHOLDER}
-                hasError={Boolean(inputError)}
-                hint={inputError}
-                disabled={analyticsActive}
+            {analyticsHandler === GOOGLE_ANALYTICS_HANDLER_TYPE.SITE_KIT && (
+              <TextInputHelperText size={TextSize.Small}>
+                <TranslateWithMarkup>
+                  {TEXT.SITE_KIT_IN_USE}
+                </TranslateWithMarkup>
+              </TextInputHelperText>
+            )}
```

Comment:
In our last meeting we discussed always displaying this text.

However, after looking at it again, I think we can simply **remove** it. It doesn't add any value in this new layout.

Reply:
I think current behaviour seems more clear from user's point of view. Because, note is necessary to convey to the user that `Site Kit` has enabled Google Analytics and will be used to track analytics.

WDYT 🤔 

| Option | Note |
| ------ | ----- |
| Use Site Kit for analytics (default) | <b>Note: </b>Site Kit by Google has already enabled Google Analytics for your Web Stories, all changes to your analytics tracking should occur there. |
| Use only Web Stories for analytics | - |
| Use both | - |

Reply:
> I think current behaviour seems more clear from user's point of view. Because, note is necessary to convey to the user that `Site Kit` has enabled Google Analytics and will be used to track analytics.

The problem is that this note does not convey this at all. The option in the dropdown does.

That's why the text needs updating and why I don't think it makes sense to show this Note only when the first option is chosen. Either always show it (if Site Kit is active), or never show it.

If we want to rephrase it still, it would make more sense to use something like:

> If Site Kit is active, it will be used to set up Google Analytics by default. However, you can customize the behavior in case you need more flexibility.

Reply:
> If Site Kit is active, it will be used to set up Google Analytics by default. However, you can customize the behavior in case you need more flexibility.

Seems to make more sense 👍🏻

Reply:
@swissspidy I have updated Site Kit note to be displayed regardless of option chosen in the analytics dropdown.

Commit Ref: https://github.com/GoogleForCreators/web-stories-wp/pull/13567/commits/b86ae3c8e819e7cbc08d056084e53b5dcf8e3ac2

Please let me know if anything else needs to be taken care of. Otherwise, the PR is ready for final review.